### ISSUE TITLE
sync mount option as default

### ISSUE THREAD

**Issue by izderadicka** [rbrito/usbmount#4]
By default usb disks are mounted with sync option. This can lead to very slow write speed (in my case almost 1000x slower then in async mode with 2.5" external USB disk). Async is also default for mount.

**Comment by mathieulj** [comment]
If you don't want sync mode, simply disable it in the configuration file. There is a very valid use case to having it enabled as USB disks can be removed unexpectedly so I don't see a need to change the default.

**Comment by rbrito** [comment]
Exactly, @mathieulj. I'm closing this bug, then, in the absence of better options, OK. If there are any other comments, I may reopen this issue.

**Comment by izderadicka** [comment]
sync is valid, no arguing, my point is that async is just more common - if you look how usb disks are auto mounted on desktop it's async.   mount command is also using async as default.  
Look at it from point of view of regular user -   I connected disc and it's very slow.  Then I mounted manually and it works OK - so my first though is something is wrong with usbmount.  I have to dig into details of mount configuration, understand sync and async options and change configuration. Sync option for my usb disk (which works otherwise fine) makes it so slow, that it's practically unusable. So even if I was concerned about data integrity, I had to use async finally.   And I think similar experience could be common for other users.    So question is if async is default elsewhere is there really good reason to have sync as default?
